lane_detection_node: 
  ros__parameters: 
    Hue_low : 69 
    Hue_high : 179 
    Saturation_low : 83 
    Saturation_high : 255 
    Value_low : 84 
    Value_high : 255 
    number_of_lines : 25 
    error_threshold : 0.15 
    Width_min : 10 
    Width_max : 148 
    inverted_filter : 0 
    camera_start_height : 106 
    camera_bottom_height : 346 
    camera_left_width : 0 
    camera_right_width : 640 
lane_guidance_node: 
  ros__parameters: 
    steering_sensitivity : 0.82 
    no_error_throttle : 0.55
    error_throttle : 0.5 
    error_threshold : 0.15 
